interactivity.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Lead a session at SPA
+layout: page
+has-nav: lead-a-session
+---
+
+<h1>How can I make my session interactive?</h1>
+<p>Every session at SPA Software in Practice is interactive in some way. To help you think about this aspect of your session here are some ways of getting people to participate. Most sessions include some presentation material as well as interactive elements.</p>
+<p>Some practical details to consider: Sessions are 75 or 150 minutes long. Room capacity is 50, with seating around tables for working in groups, and you’ll be able to present slides.</p>
```

Comment:
We still need to confirm the room capacity number(s) - I've put in 50 for now (which is in the right ballpark) instead of leaving it out.